diagnostics/diag.go

```diff
@@ -179,7 +179,7 @@ func decodeDiagJson(data []byte) (*DiagInfo, error) {
 func (d *Diagnostics) getDiagnosticFromPeers(ctx context.Context, peers map[peer.ID]int, pmes *pb.Message) (<-chan *DiagInfo, error) {
 	respdata := make(chan *DiagInfo)
 	wg := sync.WaitGroup{}
-	for p, _ := range peers {
+	for p := range peers {
```

Comment:
> I was wondering why https://github.com/ipfs/go-ipfs#install says 1.4+ while travis build has 1.3.

I can't recall anything in 1.3 failing-- @whyrusleeping ?

In general, for now, i try to get people to use a compiling environment same to ours to avoid confusion. but maybe in this case we could lower the requirement to `Go 1.3+`